lib/requirefy.js

```diff
@@ -93,12 +97,17 @@ module.exports = function requirefy(code, options, resolvePath) {
 
         break;
     }
+
+    if (moveUseStrict)
```

Comment:
Just do things as you see fit, I'll comment if I don't like certain stylings, but generally look around the project to get an idea what style I prefer.